packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md

```diff
@@ -1,35 +1,40 @@
 ---
-title: Connect your SCIM API service to Okta
-excerpt: Use Okta to make SCIM requests to your SCIM API service by creating and configuring an Okta SCIM integration from the Admin Console.
+title: Add a private SCIM integration
+excerpt: Create a private SCIM integration instance. Configure and check the attributes and their corresponding mappings in the Admin Console.
 meta:
   - name: description
-    content: Use Okta to make SCIM requests to your SCIM API service by creating and configuring an Okta SCIM integration from the Admin Console.
+    content: Use Okta to make SCIM requests to your SCIM API service by creating and configuring a private SCIM integration instance from the Admin Console.
 layout: Guides
 ---
 
-This guide teaches you how to configure Okta to make SCIM requests to your SCIM API service.
+Use this guide to create and test a private SCIM integration from your Okta org to your SCIM API service. To test and submit your SCIM integration to the Okta Integration Network (OIN) for public use, see [Submit an integration with the OIN Wizard](/docs/guides/submit-oin-app/scim/main/).
 
 ---
 
 #### Learning outcomes
 
-* Create an Okta SCIM integration.
-* Configure SCIM provisioning.
-* Verify user profile and attribute mappings.
+* Learn how to add a private SCIM integration to your Otka org
+* Learn how to configure SCIM provisioning and verify user profile and attribute mappings.
 
 #### What you need
 
-* A SCIM API service
+* A [SCIM API service](/docs/guides/scim-provisioning-integration-prepare/)
 * An [Okta Developer Edition org](https://developer.okta.com/signup/)
 
 ---
 
-## Getting connected
+## Overview
 
-Make sure that your SCIM implementation passes all the runscope tests before integrating it with Okta. For more information on testing your SCIM implementation, see [Prepare your SCIM API service](/docs/guides/scim-provisioning-integration-prepare/main/#test-your-scim-api)
+The instructions in this guide show you how to create a private SCIM integration instance in Okta to test your SCIM service. This integration instance is considered private because it's only available in the org from where the integration instance was created.
```

Comment:
```suggestion
The instructions in this guide show you how to create an instance of a private SCIM integration in Okta so you can test your SCIM service. This integration is considered private because it's only available in the org from where the integration instance was created.
```